pkgs/development/python-modules/python_docs_theme/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchFromGitHub, sphinx }:
+
+buildPythonPackage rec {
+  pname = "python_docs_theme";
+  version = "2023.3.1";
+  format = "flit";
+
+  src = fetchFromGitHub {
+    owner = "python";
+    repo = "python-docs-theme";
+    rev = "refs/tags/${version}";
```

You should be able to just put version here:

```suggestion
    rev = version;
```

---

I prefer explicit version, but okay.

---

Applied.